src/parsers/micro-rdfa-parser.js

```diff
@@ -45,10 +46,14 @@ const createHandler = function (specName) {
   const { TYPE, PROP, REQU } = getAttrNames(specName);
 
   const onopentag = function (tagName, attribs) {
+    if (tagName === "meta" && "itemtype" in attribs) {
```

Comment:
I think we can leave delete here, but I'd add another condition at the end: `... && specName.toLowerCase().startsWith("micro")` to make sure we only do this when parsing microdata.